sdk/python/kfp/_client.py

```diff
@@ -294,7 +294,7 @@ def list_pipelines(self, page_token='', page_size=10, sort_by=''):
     """
     return self._pipelines_api.list_pipelines(page_token=page_token, page_size=page_size, sort_by=sort_by)
 
-  def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, params={}, pipeline_id=None):
+  def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, params={}, pipeline_id=None, namespace=None):
```

Comment:
There are other functions that can run a pipeline. Do you want to add namespace to the other ones? like: create_run_from_pipeline_func

Reply:
Done.